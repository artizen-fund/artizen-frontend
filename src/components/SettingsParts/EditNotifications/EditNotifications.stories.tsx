import EditNotifications from './'

export default {
  title: 'components/EditNotifications',
  component: EditNotifications,
  argTypes: {},
}

export const EditNotificationsComponent = (props: any) => <EditNotifications {...props} />
