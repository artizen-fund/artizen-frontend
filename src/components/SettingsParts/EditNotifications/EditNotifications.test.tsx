import renderer from 'react-test-renderer'
import EditNotifications from './'

describe('EditNotifications', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<EditNotifications />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
