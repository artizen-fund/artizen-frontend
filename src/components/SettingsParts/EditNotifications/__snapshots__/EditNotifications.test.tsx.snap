// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditNotifications renders correctly 1`] = `
<div
  className="sc-f28797cd-0"
>
  <div
    className="vertical-layout"
    hidden={false}
  >
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 faiSBs"
        id="#/properties/someString"
      >
        <div
          className="sc-3d47d0c3-0 dNJokE"
        >
          <input
            minLength={2}
            onBlur={[Function]}
            onChange={[Function]}
            placeholder=" "
            required={false}
            type="text"
            value="Some default value"
          />
          <label
            className="sc-aacc9954-0 dyxLqB"
          >
            Tell us a thing.
          </label>
          <div
            className="sc-56d3ceba-0 dWOypg"
          >
            <div
              className="sc-3ba9b100-0 fGdMTk"
              color="night"
            />
          </div>
        </div>
        <div
          className="sc-a6becebf-0 iqYcJk"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 idfaAV"
        id="#/properties/someEmail"
      >
        <div
          className="sc-3d47d0c3-0 dNJokE"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            placeholder="your@email.com"
            required={true}
            type="text"
            value=""
          />
          <label
            className="sc-aacc9954-0 dyxLqB"
          >
            Email Address
             *
          </label>
          <div
            className="sc-56d3ceba-0 dWOypg"
          >
            <div
              className="sc-3ba9b100-0 fGdMTk"
              color="night"
            />
          </div>
        </div>
        <div
          className="sc-a6becebf-0 iqYcJk"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <div
        className="sc-2f9d5146-0 iXKXvK"
        id="#/properties/someNumber"
      >
        <div
          className="sc-3d47d0c3-0 dNJokE"
        >
          <input
            className="noData"
            onBlur={[Function]}
            onChange={[Function]}
            placeholder=" "
            required={false}
            step="any"
            type="number"
          />
          <label
            className="sc-aacc9954-0 dyxLqB"
          >
            What is your lucky number?
          </label>
          <div
            className="sc-56d3ceba-0 dWOypg"
          >
            <div
              className="sc-3ba9b100-0 hGSOqt"
              color="night"
            />
          </div>
        </div>
        <div
          className="sc-a6becebf-0 iqYcJk"
        />
      </div>
    </div>
    <div
      className="vertical-layout-item"
    >
      <label
        className="sc-ba7c8cb0-4 icOoZR"
        id="#/properties/someBoolean"
      >
        <div
          className="sc-ba7c8cb0-0 drqsMz"
        >
          <input
            checked={true}
            className="sc-ba7c8cb0-1 gDeSTk"
            disabled={false}
            onChange={[Function]}
            required={true}
            type="checkbox"
          />
          <span
            checked={true}
            className="sc-ba7c8cb0-2 drHdUG"
          >
            <div
              className="sc-3ba9b100-0 kxhsMF"
            />
          </span>
        </div>
        <span
          className="sc-ba7c8cb0-3 brhBsN"
        >
          Do you like to click checkmarks?
        </span>
      </label>
    </div>
  </div>
  <button
    className="sc-313ded62-2 htpSLU undefined  disabled"
    disabled={true}
    onClick={[Function]}
  >
    <span>
      Submit
    </span>
  </button>
</div>
`;
